### PR TITLE
Create category and parent category and downloading slug package

### PR DESCRIPTION
**Pull request by afnanali1992**

### CODE REVIEW

File: database/migrations/2018_04_28_200006_create_categories_table.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('categories', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name'); 
+            $table->string('slug')->nullable();;;
```
Comment (ShaikhahMaasher) [inline database/migrations/2018_04_28_200006_create_categories_table.php]: @afnanali1992 Please remove extra semicolons here. 
** Check also desc field.

File: database/migrations/2018_04_28_200053_create_parent_categories_table.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('parent_categories', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
+            $table->text('desc')->nullable();;
```
Comment (ShaikhahMaasher) [inline database/migrations/2018_04_28_200053_create_parent_categories_table.php]: Extra semicolons!

File: resources/views/admin/categories/create-parent.blade.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
+@extends('shop.shop_layouts.master')
```
Comment (ShaikhahMaasher) [inline resources/views/admin/categories/create-parent.blade.php]: @afnanali1992 Why extending front-end layout? Please make it extending the backend.

File: resources/views/admin/categories/index.blade.php [hunk 4]
```diff
@@ -0,0 +1,59 @@
+@extends('shop.shop_layouts.master')
+@section('content')
+<h3>List of Categories</h3>
+<hr>
+<div class="panel panel-default">
+  <!-- Default panel contents -->
+  <div class="panel-heading">Panel heading</div>
+  <div class="panel-body">
+      <div class="col-sm-6">
```
Comment (ShaikhahMaasher) [inline resources/views/admin/categories/index.blade.php]: @afnanali1992 Where's the div close tag here??
Please check all syntax and use pretty-css to make the code look nicer, now it's hard to read!

File: database/migrations/2018_04_28_200006_create_categories_table.php (hunk 1 above)
Comment (afnanali1992) [reply database/migrations/2018_04_28_200006_create_categories_table.php]: Ok, I removed them, what about desc ?!

**Pull request merged**